### PR TITLE
Update documentation of Bondi-Michel accretion

### PR DESCRIPTION
**Pull request by nilsdeppe**
## Proposed changes

- Add references
- Clean up notation
- Add quantities at the horizon as a function of the values specified in the input file.

### Types of changes:

- [ ] Bugfix
- [x] New feature

### Component:

- [ ] Code
- [x] Documentation
- [ ] Build system
- [ ] Continuous integration

### Code review checklist

- [ ] The PR passes all checks, including unit tests, `clang-tidy` and `IWYU`.
  For instructions on how to perform the CI checks locally refer to the [Dev
  guide on the Travis CI](https://spectre-code.org/travis_guide.html).
- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).

### CODE REVIEW

File: src/PointwiseFunctions/AnalyticSolutions/GrMhd/BondiMichel.hpp [hunk 1]
```diff
@@ -23,42 +23,146 @@ namespace grmhd {
...
  *
- * An analytic solution to the 3-D GrMhd system. The user specifies the sonic
+ * An analytic solution to the 3-D GRMHD system. The user specifies the sonic
  * radius \f$r_c\f$ and sonic rest mass density \f$\rho_c\f$, which are the
  * radius and rest mass density at the sonic point, the radius at which the
  * fluid's Eulerian velocity as seen by a distant observer overtakes the local
- * sound speed \f$c_{s,c}\f$. With a specified polytropic exponent \f$\Gamma\f$,
+ * sound speed \f$c_{s,c}\f$. With a specified polytropic exponent \f$\gamma\f$,
  * these quantities can be related to the sound speed at infinity
  * \f$c_{s,\inf}\f$ using the following relations:
```
Comment (carmaza) [inline src/PointwiseFunctions/AnalyticSolutions/GrMhd/BondiMichel.hpp]: \infty

File: src/PointwiseFunctions/AnalyticSolutions/GrMhd/BondiMichel.hpp [hunk 2]
```diff
@@ -23,42 +23,146 @@ namespace grmhd {
...
  * an appearance and one must take care to bracket the correct root. This is
- * done by using the upper bound \f$\frac{\dot{M}}{4\pi}\sqrt{\frac{2}{Mr^3}}\f$
+ * done by using the upper bound
+ * \f$\frac{\dot{M}}{4\pi}\sqrt{\frac{2}{Mr^3}}\f$.
  *
- * Additionally specified by the user are the polytropic exponent \f$\Gamma\f$,
- * and the strength parameter of the magnetic field \f$B\f$.
+ * Additionally specified by the user are the polytropic exponent \f$\gamma\f$,
+ * and the strength parameter of the magnetic field \f$B_0\f$.
  * In Kerr-Schild-Cartesian coordinates \f$(x, y, z)\f$, where
```
Comment (carmaza) [inline src/PointwiseFunctions/AnalyticSolutions/GrMhd/BondiMichel.hpp]: In Cartesian Kerr-Schild coordinates, so it's consistent with the \brief at the beginning.

File: src/PointwiseFunctions/AnalyticSolutions/GrMhd/BondiMichel.hpp [hunk 3]
```diff
@@ -23,42 +23,146 @@ namespace grmhd {
...
+ * \brief Bondi-Michel accretion \cite Michel1972 with superposed magnetic field
+ * in Schwarzschild spacetime in Cartesian Kerr-Schild coordinates.
  *
- * An analytic solution to the 3-D GrMhd system. The user specifies the sonic
+ * An analytic solution to the 3-D GRMHD system. The user specifies the sonic
  * radius \f$r_c\f$ and sonic rest mass density \f$\rho_c\f$, which are the
  * radius and rest mass density at the sonic point, the radius at which the
  * fluid's Eulerian velocity as seen by a distant observer overtakes the local
- * sound speed \f$c_{s,c}\f$. With a specified polytropic exponent \f$\Gamma\f$,
+ * sound speed \f$c_{s,c}\f$. With a specified polytropic exponent \f$\gamma\f$,
```
Comment (fmahebert) [inline src/PointwiseFunctions/AnalyticSolutions/GrMhd/BondiMichel.hpp:33]: In the documentation of the equations of state, I see:
- `IdealFluid.hpp`: gamma for the adiabatic index
- `PolytropicFluid.hpp`: Gamma for the polytropic exponent

Why make this change away from consistency?

Comment (nilsdeppe) [reply src/PointwiseFunctions/AnalyticSolutions/GrMhd/BondiMichel.hpp:33]: We need to move everything to the notation we are using in the paper (sounds like a fun task, right? :P ). So I figured I'd start here.

**Comment by nilsdeppe** [comment]
@carmaza @gsb76 I've rebased this on develop and addressed comments. Could you please let me know if there's anything else? Thanks!

**Review by mar-celine — APPROVED** [review]

**Review by carmaza — APPROVED** [review]

**Comment by kidder** [comment]
ignoring test time out

**Pull request merged**